toastedmarshmallow/jit.py

```diff
@@ -61,15 +54,40 @@ def attr_str(attr_name):
 class FieldSerializer(object):
     """Base class for generating code to serialize a field.
     """
+    def __init__(self, context=None):
+        # type: (JitContext) -> None
```

Comment:
nit: Do you need `Optional[JitContext]` here, or does it figure it out from the default arg?